services/app-web/src/common-code/data-helpers/makeDocumentKeyLookupList.ts

```diff
@@ -0,0 +1,52 @@
+import { base64ToDomain } from '../proto/stateSubmission'
+import { HealthPlanPackage } from '../../gen/gqlClient'
+import { parseKey } from '../s3URLEncoding'
+
+type LookupListType = {
+    currentDocuments: string[]
+    previousDocuments: string[]
+}
+
+const getKey = (s3URL: string) => {
+    const key = parseKey(s3URL)
+    return key instanceof Error ? null : key
+}
+
+export const makeDocumentList = (
+    submissions: HealthPlanPackage
+): LookupListType | undefined => {
```

Comment:
Is it possible for this function to return undefied? Seems like it would always return an empty pair of arrays